DataFormats/VertexReco/interface/Vertex.h

```diff
@@ -43,19 +43,27 @@ namespace reco {
     typedef math::Error<dimension>::type Error;
```

Comment:
better defind dimension4D = 4 enum and use it instead of "dimension+1" everywhere